pandas/tests/io/test_excel.py

```diff
@@ -165,58 +165,100 @@ def setup_method(self, method):
         self.check_skip()
         super(ReadingTestsBase, self).setup_method(method)
 
-    def test_parse_cols_int(self):
-
+    def test_usecols_int(self):
+        # GH4988: inconsistent naming convention for read_excel column select
         dfref = self.get_csv_refdf('test1')
         dfref = dfref.reindex(columns=['A', 'B', 'C'])
-        df1 = self.get_exceldf('test1', 'Sheet1', index_col=0, parse_cols=3)
+        df1 = self.get_exceldf('test1', 'Sheet1', index_col=0, usecols=3)
         df2 = self.get_exceldf('test1', 'Sheet2', skiprows=[1], index_col=0,
-                               parse_cols=3)
-        # TODO add index to xls file)
+                               usecols=3)
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            df3 = self.get_exceldf('test1', 'Sheet2', skiprows=[1],
+                                   index_col=0, parse_cols=3)
+
         tm.assert_frame_equal(df1, dfref, check_names=False)
         tm.assert_frame_equal(df2, dfref, check_names=False)
+        tm.assert_frame_equal(df3, dfref, check_names=False)  # backward compat
 
-    def test_parse_cols_list(self):
-
+    def test_usecols_list(self):
+        # GH4988: inconsistent naming convention for read_excel column select
         dfref = self.get_csv_refdf('test1')
         dfref = dfref.reindex(columns=['B', 'C'])
         df1 = self.get_exceldf('test1', 'Sheet1', index_col=0,
-                               parse_cols=[0, 2, 3])
+                               usecols=[0, 2, 3])
         df2 = self.get_exceldf('test1', 'Sheet2', skiprows=[1], index_col=0,
-                               parse_cols=[0, 2, 3])
-        # TODO add index to xls file)
+                               usecols=[0, 2, 3])
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+            df3 = self.get_exceldf('test1', 'Sheet1', index_col=0,
+                                   parse_cols=[0, 2, 3])
+
         tm.assert_frame_equal(df1, dfref, check_names=False)
         tm.assert_frame_equal(df2, dfref, check_names=False)
+        tm.assert_frame_equal(df3, dfref, check_names=False)  # backward compat
 
-    def test_parse_cols_str(self):
```

Comment:
leave the original tests structure (sure you can change the name to conform), but don't change the tests (in THIS PR).